kfdef/kfctl_caasp.yaml

```diff
@@ -112,39 +112,17 @@ spec:
       - name: userid-header
         value: kubeflow-userid
       - name: oidc_provider
-        value: http://dex.auth.svc.cluster.local:5556/dex
+        value: https://dex.replace.me:32000
```

Comment:
IMO this is fine for now. it would be nice to be able to template these out somehow as to not require the editing of the file to deploy it on caasp. (unless i'm miss understanding)

Reply:
I think these are meant to be edited. Otherwise I can't imagine what they're doing with static_password_hash on line 143.